patches/net/minecraft/world/item/crafting/RecipeManager.java.patch

```diff
@@ -9,7 +17,15 @@
          );
          List<RecipeHolder<?>> list = new ArrayList<>(sortedmap.size());
          sortedmap.forEach((p_379232_, p_379233_) -> {
-@@ -260,6 +_,11 @@
+@@ -93,6 +_,7 @@
+             .stream()
+             .map(p_380840_ -> new RecipeManager.IngredientCollector(p_380840_.getKey(), p_380840_.getValue()))
+             .toList();
++        this.recipes.order(this.recipePriorities); // Neo: order recipes by priority.
```

Comment:
I still think it would be nicer at the top of the method. Conceptually the other calls above are part of the iteration, and we want to reorder before any iteration.